lib/web_ui/lib/src/engine/html/path/path.dart

```diff
@@ -1310,7 +1310,8 @@ class SurfacePath implements ui.Path {
     pathRef.startEdit();
     final int pointCount = pathRef.countPoints();
     final Float32List points = pathRef.points;
-    for (int i = 0, len = pointCount * 2; i < len; i += 2) {
+    final int len = pointCount * 2;
+    for (int i = 0; i < len; i += 2) {
```

Comment:
Is there anything I can do to fix it? I'm not sure which part you are referring to.

Reply:
Nope, I don't think there's anything that can be done to fix this.

I meant having to split the initialization of `len` outside of the `for` loop because it can't be defined as `final` inside it.